src/features/DebugSession.ts

```diff
@@ -358,7 +358,12 @@ export class PickPSHostProcessFeature implements IFeature {
 
     private pickPSHostProcess(): Thenable<string> {
         return this.languageClient.sendRequest(GetPSHostProcessesRequestType, null).then((hostProcesses) => {
-            const items: IProcessItem[] = [];
+            // Start with the current PowerShell process in the list.
+            const items: IProcessItem[] = [{
+                label: "Current",
+                description: "The current PowerShell process.",
```

Comment:
Will this be obvious to users with both PowerShell terminals and PSIC?  I wonder if we should be more specific in the description e.g. `The current PowerShell Integrated Console` or something like that.

Reply:
```suggestion
                description: "The current PowerShell Integrated Console process.",
```